source/_integrations/brightsky.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Bright Sky"
+description: "Instructions on how to integrate Bright Sky within Home Assistant."
+logo: dwd.png
+featured: false
+ha_category:
+  - Weather
+ha_release: 0.114
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
+ha_codeowners:
+  - '@mweinelt'
+ha_domain: brightsky
+---
+
+The `brightsky` platform uses the [Bright Sky](https://brightsky.dev) JSON API to query meteorological provided by the Deutsche Wetterdienst (DWD)
+
+One limitation of this integration stems from the DWDs focus on Germany and the Bright Sky maintainers state the following:
+>  Due to its nature as German meteorological service, the observations published by the DWD have a strong focus on Germany. While there is data for some locations outside of Germany, mainly in the rest of Europe, it is currently not a priority for us. Your mileage may vary.
+
+## Configuration
+
+The API currently imposes no rate limits, this platform updates at most every 10 minutes.
+
+To add Bright Sky to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: brightsky
```

Comment:
We no longer accept platform integrations. Furthermore, integrations that are connecting to a device or service need to have a config flow in the UI.